### ISSUE TITLE
Add MedImg for streaming

### ISSUE THREAD

**Issue by lhallee** [huggingface/datasets#6912]
### Feature request

Host the MedImg dataset (similar to Imagenet but for biomedical images). 

### Motivation

There is a clear need for biomedical image foundation models and large scale biomedical datasets that are easily streamable. This would be an excellent tool for the biomedical community.

### Your contribution

MedImg can be found [here](https://www.cuilab.cn/medimg/#). I assume there are standard pipelines for processing and getting onto Huggingface Hub, but I could provide a zipped version of the 200gb dataset if needed.

**Comment by lhallee** [comment]
@mariosasko, @lhoestq, @albertvillanova
Hello! Can anyone help? or can you guys suggest who can help with this?

**Comment by lhoestq** [comment]
Hi ! Feel free to download the dataset and create a `Dataset` object with it.

Then your'll be able to use `push_to_hub()` to upload the dataset to HF in Parquet format and make it streamable :)

**Comment by lhallee** [comment]
> Hi ! Feel free to download the dataset and create a `Dataset` object with it.
> 
> Then your'll be able to use `push_to_hub()` to upload the dataset to HF in Parquet format and make it streamable :)

The dataset is several TB in total, which I do not have the resources to handle.

**Comment by lhallee** [comment]
Hi @lhoestq and @albertvillanova , just following up about this.

**Comment by lhallee** [comment]
Sure, that makes sense. However, isn't there a size limit to what typical users can push?

**Comment by lhoestq** [comment]
Yes there is a limit, simply let us know by email at datasets [at] huggingface.co - this way we can also help making sure the dataset is all good for people to use it easily

**Comment by lhallee** [comment]
> Yes there is a limit, simply let us know by email at datasets [at] huggingface.co - this way we can give you a storage grant also help making sure the dataset is all good for people to use it easily

Got it, that would be great.
